readthedocs/templates/projects/project_version_list.html

```diff
@@ -22,6 +22,21 @@
     <div class="module project-versions-active">
       <div class="module-wrapper">
 
+        {% block resync_versions %}
+          {% if request.user|is_admin:project %}
+          <div class="button-bar">
+            <ul>
+              <li>
+                <form method="post" action="{% url 'project_repository_resync' project_slug=project.slug %}">
+                  {% csrf_token %}
+                  <input type="submit" value="{% trans 'Re-sync Versions' %}">
```

Comment:
I'd use "Sync Versions" for the label of the button.

```suggestion
                  <input type="submit" value="{% trans 'Sync Versions' %}">
```